### DISCUSSION TITLE
4.0.7

### DISCUSSION THREAD

**Discussion by dharmesh-hemaram** [Dhruv-Techapps/acf-docs#386] [Release]
## What's Changed
* Feature/algolia by @dharmesh-hemaram in https://github.com/Dhruv-Techapps/acf-docs/pull/383
* Added icons by @dharmesh-hemaram in https://github.com/Dhruv-Techapps/acf-docs/pull/384
* fav icons by @dharmesh-hemaram in https://github.com/Dhruv-Techapps/acf-docs/pull/385

**Full Changelog**: https://github.com/Dhruv-Techapps/acf-docs/compare/4.0.4...4.0.7

<hr /><em>This discussion was created from the release <a href='https://github.com/Dhruv-Techapps/acf-docs/releases/tag/4.0.7'>4.0.7</a>.</em>

**Comment by acls2x** [comment]
Sorry was not sure where to ask this or how to create a new discussion    The invite to discord on your website says invalid so i am not sure how to get into the discord group  
Lets say I have recorded inputs on a single web page  in two configurations     so my inputs are different for each configuration Is there a way to get only one profile to run I know one way is to set it to manual mode and use hotkeys or you could disable one of the profiles    It would be nice to have a menu where I could select the profile I wished to be run

**Reply by dharmesh-hemaram** [reply]
Can you explain more why you require two different configuration . may be I could provide you solution for same with single configuration.

**Reply by acls2x** [reply]
I may fill out a form 4 times in one day where 75% of the fills are the same for each time, but  each form will have some unique inputs I know by reviewing records pertinent to each case There can be 50-100 baseline items to fill (check boxes radiao buttons or text boxes
 It is not the same input that changes each time so I would like to have a general fill form and then edit it specific to the case I am seeing and save it to fill the actual form at the beginning of the case The specifics change on a weekly basis

**Reply by dharmesh-hemaram** [reply]
> I may fill out a form 4 times in one day where 75% of the fills are the same for each time, but  each form will have some unique inputs I know by reviewing records pertinent to each case There can be 50-100 baseline items to fill (check boxes radiao buttons or text boxes
> 
>  It is not the same input that changes each time so I would like to have a general fill form and then edit it specific to the case I am seeing and save it to fill the actual form at the beginning of the case The specifics change on a weekly basis
> 
> 

Can you provide some examples to try out

**Reply by acls2x** [reply]
Not in html form as  my forms require secure sign in..though i will keep looking for html examples of something similar The below PDF is quite similar

https://www.fmcsa.dot.gov/sites/fmcsa.dot.gov/files/2023-12/Medical%20Examination%20Report%20Form%20MCSA-5875%2023.pdf

**Comment by RV-LN** [comment]
Hi Dharmesh
Nice to meet you, I just saw you develop auto click auto fill template, but I’m a very new user on this stuff. I just would like you to help me to create the autofill on online shopping website call “ [www.shopee.co.th] (http://www.shopee.co.th/)” I would like to buy the product which is not available now but I want to use autofill to refresh product loop until available and then add more item also after that go to cart and make and order. Is that possible for you to help me on this ?

**Reply by dharmesh-hemaram** [reply]
yes its possible . you can make use of action settings retry to keep trying until the product is available also it can do additional check on page that if the price is above or below specific amount
